pytest_responses.py

```diff
@@ -1,8 +1,17 @@
+from distutils.version import LooseVersion
+
 import pytest
 
 import responses as responses_
 
 
+def get_withoutresponses_marker(item):
+    if LooseVersion(pytest.__version__) >= LooseVersion('4.0.0'):
+        return item.get_closest_marker('withoutresponses')
+    else:
+        return item.get_marker('withoutresponses')
```

Comment:
I thought about just checking for the `get_closest_marker` attribute as well. Open to suggestions